features/networking/pod.feature

```diff
@@ -297,3 +297,45 @@ Feature: Pod related networking scenarios
       | type          | merge                                                         						   |
     Then the step should fail
     And the output should contain "endpoints "<%= cb.ping_pod.name %>" is forbidden: endpoint port TCP:22623 is not allowed"
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-21846
+  @admin
+  @destructive
+  Scenario: ovn pod can be scheduled even if the node taint to unschedule
+    Given the env is using "OVNKubernetes" networkType
+    And I store all worker nodes to the :nodes clipboard
+    #Tainting node to NoSchedule
+    When I run the :oadm_taint_nodes admin command with:
+      | node_name | <%= cb.nodes[0].name %> |
+      | key_val   | key2=value2:NoSchedule  |
+    Then the step should succeed
+    And I register clean-up steps:
+    """
+    I run the :oadm_taint_nodes admin command with:
+      | node_name | <%= cb.nodes[0].name %> |
+      | key_val   | key2-  |
+    Then the step should succeed
+    """
+    Given I have a project
+    #Makng sure test pods can' be sceduled on tha tainted node
+    And I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/pod-for-ping.json |
+    Then the step should succeed
+    And the pod named "hello-pod" status becomes :pending
```

Comment:
How to make sure this hello-pod will be created in the tainted node not the other running node?

Reply:
Good catch @weliang1 Thanks. It also needs [spec][nodename].

Reply:
@weliang1 Made it more optimized. All nodes are tainted now in the beginning so no need to check which node hello-pod is being created on. Also added 60 sec timer for hello-pod in case it doesn't reach to pending state for 60 seconds,we should fail and know something is wrong. And then we check ovnkube pod should be deleted/recreated on a worker node.

Reply:
OCP-21846 LGTM now